conn/physic/units.go

```diff
@@ -67,6 +158,9 @@ const (
 	Theta  Angle = 6283185307 * NanoRadian
 	Pi     Angle = 3141592653 * NanoRadian
 	Degree Angle = 17453293 * NanoRadian
+
+	maxAngle = 9223372036854775807 * NanoRadian
```

Comment:
I wonder if these should be rewritten as:
```
maxAngle = (1<<63)*NanoRadian
```
then in the test with:
- as passing, maxAngle-1
- as failing, maxAngle